index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html>
+  <head bgcolor="#B36666">
+      <title>My Test Page </title>
+  </head>
+  <body bgcolor="#FFCCCC">
+	 <section id="Header">
+	    <Header>
+	    	<h1><center>Stephen Gant</center></h1>
+	    	<center><IMG SRC="picture1.jpg" ALT="A picture of my dog and me" WIDTH=400 ></center>
+	     	<h3><center>I can program..... <i>kinda</i></center></h3>
+	    </Header> 
+	  </section>
+	  <nav>
+    	<ul>
+    		<li><a href="#Header">Return to Top</a></li>
+    		<li><a href="#About Me">About Me</a></li>
+    		<li><a href="#Schedule">My Sclass Scedule This Semester</a></li>
+    	</ul>
+       </nav>
+       <hr />
+      <Section id="About Me">
+	      <p>
+	      	<h2>Some things about me:<h2>
+	      </p>
+	      <h3>Life Skills</h3>
+	      	<ul>
+	        	<li><b>I can make bold text</b></li>
+	        	<li>I can also code html</li>
+	          		<ul>
+	            		<li style="color: cyan;"><u>Which is pretty cool... just like this text</u></li>
+	          		</ul>
+	      		<li> <a href="http://images.rapgenius.com/205030145a1abcaed4efe27133eb23c6.340x255x61.gif">My philosophy on life</a></li>
+	      		<li><a href="anotherpage.html">link</a></li>
+	      	</ul>
+      </Section>
+      <section id="Schedule">
+      	<table style="width:30%">
+      		<tr>
+      			<td><h3><b><center>Course</td>
```

Comment:
These `<b>` and `<center>` tags are missing their respective `</b>` and `</center>` tags. My browser understood what you meant, but my editor sure didn't!
![screen shot 2015-09-09 at 11 48 12 am](https://cloud.githubusercontent.com/assets/2058614/9766812/e8d4cbbe-56e8-11e5-8571-b01d5c1b2c3e.png)